addons/notes/src/Panel.tsx

```diff
@@ -56,7 +56,9 @@ export const SyntaxHighlighter = ({ className, children, ...props }: SyntaxHighl
   // className: "lang-jsx"
   const language = className.split('-');
   return (
-    <SyntaxHighlighterBase language={language[1] || 'plaintext'} bordered copyable {...props}>{children}</SyntaxHighlighterBase>
```

Comment:
Why it is here? 🤔

Reply:
Not Exactly sure why and I’d have to look at it to figure it out but it looks like a format change

Reply:
Just a formatting change, I guess it was a warning as didn't cause the build to fail before, but got fixed by accident/automatically